src/DependencyInjection/Configuration.php

```diff
@@ -46,6 +46,7 @@ public function getConfigTreeBuilder()
                 ->variableNode('stats')->end()
                 ->scalarNode('endpoint')->end()
                 ->variableNode('endpoint_discovery')->end()
+                ->scalarNode('handler')->end()
```

Comment:
I am not sure if the option should just be called "handler" or "http_handler", either option key is actually allowed to be used for the sdk. However in order to reduce confusion as to why this option isn't in the "http" arrray I went with just `handler` as a name.

Reply:
The `handler` and the `http_handler` options are different things in the SDK.  The `handler` option takes a command object and returns a promise with the value sent back by the service.  The `http_handler` is similar, but the input is a PSR-7 request object instead.  This means the `handler` comes into play earlier and is what forms the request, the `http_handler` is used after the request is fully formed.  

Which of these functionalities are you hoping to enable?

Reply:
Ah, I see, thanks for the explanation!

In our case we would want to overwrite the http_handler, however I guess giving the option to overwrite either would probably be helpful to the consumers of this package. I can add both options in this PR unless you see that either option isn't "recommended" or anything.